README.md

```diff
@@ -15,14 +15,22 @@
 
 ### Compiling from Source
 
-If you prefer to compile from source or are contributing to the project, you can install `kuadrantctl` using `go install`. This method requires Golang 1.21 or newer.
```

Comment:
I would remove the suggestion to install using `go install`. `make install` is the only go option